src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -233,7 +233,7 @@ def get_event_stats_metrics(_, user_query, params, rollup, zerofill_results, __)
             top_events = get_top_events(
                 user_query=user_query,
                 params=params,
-                event_limit=TOP_EVENTS_LIMIT,
+                event_limit=int(request.GET.get("topEvents", DEFAULT_TOP_EVENTS_LIMIT)),
```

Comment:
Not a blocker, but: do we need a maximum here, or will Snuba take care of that downstream? In general having unbounded user-controlled sizes makes me a touch worried about the load a person can generate with one request.

Reply:
added top limit of 1000